docs/sources/alerting/_index.md

```diff
@@ -21,6 +21,8 @@ You can create and edit alerting rules for Grafana managed alerts, Cortex alerts
 
 As part of the new alert changes, we have introduced a new data source, Alertmanager, which includes built-in support for Prometheus Alertmanager. It is presently in alpha and it not accessible unless alpha plugins are enabled in Grafana settings. For more information, refer to [Alertmanager data source]({{< relref "../datasources/alertmanager.md" >}}).
 
+> **Note:** Currently the Alertmanager configuration is shared across organisations. Therefore users are advised to use the new Grafana 8 Alerts only if they have one organisation otherwise all contact points, notification policies, silences and templates will be visible by all organizations.
+
```

Comment:
Wonder if we should clarify that this only applies to the built-in alertmanager, and how will the user know that they are using the built in vs external?

Reply:
I will try to rephrase it to make it more clear.